.github/workflows/integration.yml

```diff
@@ -75,6 +75,10 @@ jobs:
         run: |
           poetry run pip install numpy
           poetry run pip install pygdal==3.0.4.10
+      - name: Install pdfkit
+        working-directory: ./api
+        run: |
+          poetry run pip install pdfkit
```

Comment:
Why isn't poetry installing this?

Reply:
Poetry does install this but there's an OS dependency I need to add, this line is incorrect, should be installed with `apt`

Reply:
Gross. It's a pity it isn't pure python. I'd rather not have an binary dependencies, but just find a pdf library that actually does the job has been so hard!